### ISSUE TITLE
4.16.1 HA Kubernetes not deploying multiple master nodes

### ISSUE THREAD

**Issue by AWrightforge** [apache/cloudstack#6321]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
HA kubernetes 
~~~

##### CLOUDSTACK VERSION

~~~
4.16.1
~~~

##### CONFIGURATION

KVM, advanced networking zone

##### OS / ENVIRONMENT

ubuntu 20.04 

##### SUMMARY

CS passes >2 to the plugin and then right after the first control node is deployed, it starts deploying a worker node.

##### STEPS TO REPRODUCE

~~~
Deploy a HA kubernetes cluster and watch only 1 control node actually deploys
~~~

##### EXPECTED RESULTS

~~~
I expect to see multiple control nodes in a HA K8s cluster
~~~

##### ACTUAL RESULTS

~~~

~~~

**Comment by Pearl1594** [comment]
@AWrightforge this issue may most likely be caused due to specific services starting in the nodes prior to creation of those specific service files during bootstrapping of the nodes, as explained in https://github.com/apache/cloudstack/issues/5999. This has been addressed in 4.17(main). It would be great if you could verify if your issue is fixed by using the main branch. Thanks

**Comment by rohityadavcloud** [comment]
It appears this is fixed in 4.17.0.0 release. Thanks for the bug-report @AWrightforge kindly test 4.17.0.0 and re-open if you're still able to reproduce this.
